invesalius/gui/task_navigator.py

```diff
@@ -1527,6 +1527,18 @@ def OnMenuSetTarget(self, evt):
         else:
             wx.MessageBox(_("No data selected."), _("InVesalius 3"))
 
+    def OnMenuRemoveTarget(self, evt):
+        idx = self.lc.GetFocusedItem()
+        self.markers[idx].is_target = False
+        self.lc.SetItemBackgroundColour(idx, 'white')
+        Publisher.sendMessage('Set target transparency', status=False, index=idx)
+        self.lc.SetItem(idx, const.TARGET_COLUMN, "")
+        Publisher.sendMessage('Disable or enable coil tracker', status=False)
+        if self.tracker.tracker_id == const.ROBOT:
+            Publisher.sendMessage('Update robot target', robot_tracker_flag=False,
+                                  target_index=None, target=None)
+        wx.MessageBox(_("Target removed."), _("InVesalius 3"))
+
```

Comment:
Using the "Debug tracker (approach)", even after removing the target, it still thinks that there is a valid target to approach. Remove this behavior.

Reply:
I worked around this issue by setting the target to None 706749e. However, the approach targeting happens inside the coregistration thread. Therefore requires to stop and start the navigation to break the approach. I don't think it is worth creating a queue to update the target or import the navigation metaclass singleton (the target is used only for the approach tracker).

Reply:
Agreed!